### PR TITLE
feat(README) : Add info over how to do the releases (AEROGEAR-9381)

### PR DESCRIPTION
**Pull request by camilamacedo86**
## Motivation
https://issues.jboss.org/browse/AEROGEAR-9381

## What
- Add/update info to the README over how to do the releases
- Remove dev tag since it is no longer required

## Why
- Have this process doc

## Checklist:

- [X] Code has been tested locally by PR requester
- [ ] Changes have been successfully verified by another team member 

## Progress

- [X] Finished task
- [ ] TODO

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -467,6 +467,30 @@ NOTE: The image/tag used from https://github.com/aerogear/mobile-security-servic
 
 NOTE: The link:./Makefile[Makefile] is implemented with tasks which you should use to work with.
 
+== Release
+
+Following the steps
+
+* Create a new tag following the http://semver.org/spec/v2.0.0.html[semver]. Following an example.
```
Comment (damienomurchu) [inline README.adoc]:
```suggestion
* Create a new tag following the http://semver.org/spec/v2.0.0.html[semver], for example:

```

File: README.adoc [hunk 2]
```diff
@@ -467,6 +467,30 @@ NOTE: The image/tag used from https://github.com/aerogear/mobile-security-servic
...
+
+[source,shell]
+----
+$ git tag -a 0.1.0 -m "version 0.1.0"
+----
+
+* Push the new tag to the upstream repository.  Following an example.
```
Comment (damienomurchu) [inline README.adoc]:
```suggestion
* Push the new tag to the upstream repository., for example:
```

File: README.adoc [hunk 3]
```diff
@@ -467,6 +467,30 @@ NOTE: The image/tag used from https://github.com/aerogear/mobile-security-servic
...
+
+[source,shell]
+----
+$ git push upstream 0.1.0
+----
+
+* Update the the link:./CHANGELOG.MD[CHANGELOG.MD] with the new release.
+
+NOTE: The image with the tag will be created and pushed to https://quay.io/repository/aerogear/mobile-security-service[quay.io/repository/aerogear/mobile-security-service] by the CI.
```
Comment (damienomurchu) [inline README.adoc]:
```suggestion
NOTE: The image with the tag will be created and pushed to the https://quay.io/repository/aerogear/mobile-security-service[mobile-security-service image hosting repository] by the CI.
```

File: README.adoc [hunk 4]
```diff
@@ -467,6 +467,30 @@ NOTE: The image/tag used from https://github.com/aerogear/mobile-security-servic
...
+
+WARNING: Do not use letters in the tag as `v`. It will not work.
```
Comment (damienomurchu) [inline README.adoc]:
```suggestion
WARNING: Do not use letters in the tag such as `v`. It will not work.
```

**Comment by damienomurchu** [comment]
Some comments left in PR. 

There is [already a section in the REAMDE](https://github.com/aerogear/mobile-security-service-operator/blob/master/README.adoc#publishing-a-new-version) covering publishing new images, and think the info in your PR should be added to that section

**Comment by camilamacedo86** [comment]
hi, @damienomurchu tks for your great review. The changes were made, could you please recheck it?

**Comment by camilamacedo86** [comment]
@damienomurchu tks for the good spots, could you recheck?

**Pull request merged**